.gitignore

```diff
@@ -3,3 +3,10 @@ yarn.lock
 /.build
 /*.xcodeproj
 /main
+.vscode/settings.json
+.vscode/ipch
+.vscode/c_cpp_properties.json
```

Comment:
These are not project specific and should be in your own global gitignore.

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c